### ISSUE TITLE
Code insights landing page design

### ISSUE THREAD

**Issue by marybelzer** [sourcegraph/about#5098]

**Comment by stephzabala** [comment]
@marybelzer I heard from Fabiana that creating the page outline was blocked (therefore blocking design) but it should be ready soon. Do you know if there is an update to the page outline?

**Comment by marybelzer** [comment]
@stephzabala You're right, I should have had this in blocked status. Updated that and will let you know as soon as I hear of any changse!

**Comment by marybelzer** [comment]
@stephzabala Page outline is ready to go and in the description! 

also here for reference: https://docs.google.com/document/d/1xZlplAFrkzQUI-_O3cHoLc2pOY7oSvOg4Ggt_bo6Mbk/edit#heading=h.w69ibme4g2xj

**Comment by stephzabala** [comment]
@marybelzer Thank you! Do you know if there's still a design review tomorrow since the copy deadline was pushed out? I can show where I'm at async if we still need it, it will most likely be fairly WIP

**Comment by marybelzer** [comment]
@stephzabala there is not! it was cancelled due to that shift. will update the description here accordingly

**Comment by stephzabala** [comment]
@marybelzer Thank you for all of the clarification!!

**Comment by marybelzer** [comment]
@stephzabala Can you add a link here to your figma file? Katy is preparing to begin this work and a preview to the designs would be helpful. Thanks! CC @katjuell

**Comment by stephzabala** [comment]
@marybelzer @katjuell Figma file has been linked in the description! Most recent design is under the page titled 'WIP'. 

FYI the structure and modules are definitely still subject to change.

**Comment by marybelzer** [comment]
Feedback from Andy (originally left in Asana, copying here for visibility):
The social proof and the demo are a bit buried. I'd consider 1) having the demo before the resources section and 2) moving the social proof up
